build-tools/packages/version-tools/src/internalVersionScheme.ts

```diff
@@ -374,10 +376,10 @@ export function bumpInternalVersion(
  */
 export function getVersionRange(
```

Comment:
This function is intended to be used with internal versions - that's why it's in the internalVersionScheme file. That's why the validateVersionScheme function is called immediately. If you want to create a "make a semver into a legacy range" function, do that in a separate place. If you want to create a new function that works with any version scheme, you can do that too but you'll need to detect the version scheme - which you can't do with these legacy versions because they're not really a different scheme.